impressum.md

```diff
@@ -0,0 +1,41 @@
+# Impressum
+
+*Angaben gemäß § 5 TMG*
+
+Brockmeyer Haardiek Kerssen Kerssen Kiesow Rosemann GbR  
+Birkhahnweg 27  
+49124 Georgsmarienhütte
+
+## Vertreten durch:  
+Timo Nogueira Brockmeyer
+
+## Kontakt:
+Telefon:  
+Fax:  
+E-Mail: info@emissions-api.org
+
+## Verantwortlich für den Inhalt nach § 55 Abs. 2 RStV:
+Timo Nogueira Brockmeyer  
+Birkhahnweg 27  
+49124 Georgsmarienhütte
+
+## Haftungsausschluss:
+
+### Haftung für Inhalte
+
+Die Inhalte unserer Seiten wurden mit größter Sorgfalt erstellt. Für die Richtigkeit, Vollständigkeit und Aktualität der Inhalte können wir jedoch keine Gewähr übernehmen. Als Diensteanbieter sind wir gemäß § 7 Abs.1 TMG für eigene Inhalte auf diesen Seiten nach den allgemeinen Gesetzen verantwortlich. Nach §§ 8 bis 10 TMG sind wir als Diensteanbieter jedoch nicht verpflichtet, übermittelte oder gespeicherte fremde Informationen zu überwachen oder nach Umständen zu forschen, die auf eine rechtswidrige Tätigkeit hinweisen. Verpflichtungen zur Entfernung oder Sperrung der Nutzung von Informationen nach den allgemeinen Gesetzen bleiben hiervon unberührt. Eine diesbezügliche Haftung ist jedoch erst ab dem Zeitpunkt der Kenntnis einer konkreten Rechtsverletzung möglich. Bei Bekanntwerden von entsprechenden Rechtsverletzungen werden wir diese Inhalte umgehend entfernen.
+
+### Haftung für Links
+
+Unser Angebot enthält Links zu externen Webseiten Dritter, auf deren Inhalte wir keinen Einfluss haben. Deshalb können wir für diese fremden Inhalte auch keine Gewähr übernehmen. Für die Inhalte der verlinkten Seiten ist stets der jeweilige Anbieter oder Betreiber der Seiten verantwortlich. Die verlinkten Seiten wurden zum Zeitpunkt der Verlinkung auf mögliche Rechtsverstöße überprüft. Rechtswidrige Inhalte waren zum Zeitpunkt der Verlinkung nicht erkennbar. Eine permanente inhaltliche Kontrolle der verlinkten Seiten ist jedoch ohne konkrete Anhaltspunkte einer Rechtsverletzung nicht zumutbar. Bei Bekanntwerden von Rechtsverletzungen werden wir derartige Links umgehend entfernen.
+
+### Urheberrecht
```

Comment:
Ok, I agree.

Reply:
what is CC-0?
So at first i just will remove this block!?

Reply:
Yep, for now I would just remove this paragraph. Removing that we still lay claim to the usage rights for this website's content.

That said, we should discuss usage rights since we actually want people to e.g. grab and use the code for one of our examples or one of the images plotting data. For this, common media licenses provided by [Creative Commons](https://creativecommons.org/) are a great choice. Among those licenses, I would recommend that we go with something like [CC-BY](https://creativecommons.org/licenses/by/2.0/), [CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/) or a combination of both. CC-0 basically states “do whatever you want with this” while CC-BY basically states “do whatever you want with this but state where you got this”.

We should discuss this in more details, but for now, let's just drop this paragraph :-)

Reply:
Just noticed that you already did :-)